php/Blocks/Controls/Range.php

```diff
@@ -61,7 +61,7 @@ public function register_settings() {
 				'name'     => 'max',
 				'label'    => __( 'Maximum Value', 'genesis-custom-blocks' ),
 				'type'     => 'number',
-				'default'  => '',
+				'default'  => 100,
```

Comment:
The Range field wasn't working unless you enter a `min` and `max`. This was even an issue in the previous editor.

So this sets `0` and `100` as the default `min` and `max`.